sdk-rs/src/websocket_program_account_subscriber.rs

```diff
@@ -0,0 +1,184 @@
+use std::pin::Pin;
+use std::sync::Arc;
+use anchor_lang::AccountDeserialize;
+use base64::{Engine as _, engine::general_purpose};
+use futures_util::Future;
+use solana_account_decoder::{UiAccountEncoding, UiAccountData};
+use solana_client::rpc_filter::RpcFilterType;
+use solana_sdk::commitment_config::CommitmentConfig;
+use solana_client::pubsub_client::PubsubClient;
```

Comment:
was there a reason to choose the blocking variant over the non-blocking one? I'd say we should use the latter